general/src/protocol/debugmeta.rs

```diff
@@ -66,55 +66,78 @@ pub struct AppleDebugImage {
     pub other: Object<Value>,
 }
 
-impl Empty for DebugId {
-    #[inline]
-    fn is_empty(&self) -> bool {
-        false
-    }
-}
+macro_rules! impl_traits {
```

Comment:
We have the `derive_from_value!` and `derive_to_value!` macros, I think those can be used.

Reply:
Those only work for `Value`, sadly.